server/v2/schema.js

```diff
@@ -0,0 +1,90 @@
+const { buildSchema } = require('graphql');
+
+module.exports = buildSchema(`
+  type Weather {
+    temperature: Float
+    pressure: Float
+    humidity: Float
+    windSpeed: Float
+    windDirection: Float
```

Comment:
Verry happy with this, it's going to be nice to have this be a lot more accurate.